test/interactive001/input

```diff
@@ -1,8 +1,8 @@
-:cs 12 xs
-:cs 18 ys
-:ps 22 maprhs
-:mw 26 isElem2
-:cs 31 p
-:cs 36 xs'
-:cs 39 x
-:cs 43 case_val
+:casesplit 12 xs
+:casesplit 18 ys
+:proofsearch 22 maprhs
+:makewith 26 isElem2
+:casesplit 31 p
+:casesplit 36 xs'
+:casesplit 39 x
+:casesplit 43 case_val
```

Comment:
I see, I guess that is a good reason indeed. 
I was just wondering whether there were things that were changed, but if it is only a readability things, then I agree.